techniques/general/G102.html

```diff
@@ -113,118 +102,98 @@ <h2>Description</h2>
+            <p>略語が元の表記を必要としない場合 (例えば、当該組織が元の表記を廃止している、又は略語がその言語の一部になっている)、説明を提供する、又は適切なら、説明を必要としない単語として略語を見なす。
             </p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-ada">
-               <h3>Example 1: ADA</h3>
-               <p>Some abbreviations have more than one meaning, and the meaning depends on the context.
-                  For example, ADA means "American Dental Association" in one context and "Americans
-                  with Disabilities Act" in another. Only the expansion relevant to the context needs
-                  to be provided.
+               <h3>事例 1: ADA</h3>
+               <p>略語の中には、複数の意味があり、文脈に依存するものがある。例えば、ADA はある文脈では「American Dental Association」を意味し、別の文脈では「Americans with Disabilities Act」を意味する。文脈に適切な元の語を提供しなければならない。
                </p>
             </section>
             <section class="example" id="example-2-english-abbreviations-for-phrases-borrowed-from-latin">
-               <h3>Example 2: English abbreviations for phrases borrowed from Latin</h3>
-               <p>In the following sentence, the explanation "for example" would be provided for "e.g.":
-                  Students participating in team sports, e.g., basketball or football, must set their
-                  schedules around team practice time.
+               <h3>事例 2: 英語の略語でラテン語由来の句に対するもの</h3>
+               <p>次の文章中において、「e.g.」には「例えば」という説明が提供される: チーム制のスポーツ (e.g. バスケットボール又はフットボール) に参加する学生は、チームの練習時間に彼らのスケジュールを合わせなければならない。
                </p>
             </section>
             <section class="example" id="example-3-abs">
-               <h3>Example 3: ABS</h3>
-               <p>Some languages (including English and Dutch) borrowed the acronym ABS (Antiblockiersystem:
-                  anti-lock brakes) from German. An explanation (anti-lock brakes) is provided, rather
-                  than the expansion
+               <h3>事例 3: ABS</h3>
+               <p>いくつかの言語 (英語及びオランダ語を含む) では、頭字語 ABS (Antiblockiersystem: アンチロックブレーキ) をドイツ語から借用している。元の表記ではなく、語の説明 (anti-lock brakes) が提供されている。
                </p>
             </section>
             <section class="example" id="example-4-acronyms-with-no-expansion">
-               <h3>Example 4: acronyms with no expansion</h3>
-               <p>Examples of acronyms which no longer have expansions include</p>
+               <h3>事例 4: もはや元の語を持たない頭文字語の例</h3>
+               <p>もはや元の語を持たない頭文字語の例</p>
                <ul>
                   
-                  <li>SIL, which used to mean Summer Institute of Linguistics, is now a name in its own
-                     right. See
-                     <a href="http://www.sil.org/sil/history.htm">SIL history</a>.
+                  <li>SIL は、Summer Institute of Linguistics を意味していたが、現在はそれ自体が名前である。
+                     <a href="http://www.sil.org/sil/history.htm">SIL の歴史</a>を参照。
```

Comment:
```suggestion
                     <a href="http://www.sil.org/sil/history.htm">SIL history</a> を参照。
```

リンクが切れていいるので、訳注として

```
上記リンクはリンク切れを起こしている。代わりに、<a href="https://www.sil.org/about/history">Our History | SIL International</a> を参照のこと。
```

というフレーズを加えておきます。